src/prerequisites/project2_2023-04-03.sql

```diff
@@ -0,0 +1,127 @@
+# ************************************************************
```

Comment:
Worth doing a fresh export to replace this one, I noticed this one has a genre table, but your code refers to genres (genres is better because its plural)